### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by anthony-iacono**
Added great stuff!!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # Git Workflow Practice
 
 Practice repo
+
+This is great stuff!!!
+More great stuff
```
Comment (nzambonivergara) [inline README.md:6]: Comment 1

Comment (nzambonivergara) [inline README.md:6]: comment 2

Comment (nzambonivergara) [inline README.md:6]: Instead of great stuff, let's say awesome stuff

**Review by nzambonivergara — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (anthony-iacono) [reply README.md:6]: Thanks for the comment.

Comment (anthony-iacono) [reply README.md:6]: Thanks for the comment again.

**Pull request merged**